packages/components/src/index.ts

```diff
@@ -1,15 +1,62 @@
-import './index.css'
+import "./index.css";
 
 // Styles
-export { StcStylesProvider, useStyles, getStyles } from './styles'
-export type { StclandStyles, TableStyles } from './styles'
+export { StcStylesProvider, useStyles, getStyles } from "./styles";
+export type { StclandStyles, TableStyles } from "./styles";
 
 // Components
-export type { TableProps } from './table'
-export { Table } from './table'
+export type { TableProps } from "./table";
+export { Table } from "./table";
 
```

Comment:
@ianmarquez  Minor thing here about the ESLINT config... is not working correctly with the eslint config file? is just that we add some rules requested by Steve, and one it is `"quotes": ["error", "single"]` located in the `config/eslint.base.json` so we can have a consistent formatting 😬